spec/index.html

```diff
@@ -899,26 +899,25 @@ <h3>Triple Terms</h3>
   <section id="graph-isomorphism">
     <h3>Graph Comparison</h3>
 
+    <p>Given a bijection |M| from [=blank nodes=] to [=blank nodes=],
+      we can extend |M| to map [=RDF terms=] to [=RDF terms=],
+      [=RDF triples=] to [=RDF triples=],
+      and [=RDF graphs=] to [=RDF graphs=],
+      as follows:
+
+    <ul>
+      <li>|M|(|iri|)=|iri| if |iri| is an [=IRI=].
+      <li>|M|(|lit|)=|lit| if |lit| is a [=literal=].
+      <li>|M|(|t|)=( |M|(|s|), |M|(|p|), |M|(|o|) ) if |t| is an [=RDF triple=] of the form ( |s|, |p|, |o| ).
+      <li>|M|(|G|)={ |M|(|t|<sub>1</sub>), ..., |M|(|t|<sub>|n|</sub>) } if |G| = { |t|<sub>1</sub>, ..., |t|<sub>|n|</sub> } is an [=RDF graph=].
+    </ul>
+
     <p id="section-graph-equality">Two
       <a>RDF graphs</a> <var>G</var> and <var>G'</var> are
       <dfn data-lt="graph isomorphism|isomorphic" data-lt-noDefault class="export">isomorphic</dfn>
       (that is, they have an identical form)
```

Comment:
```suggestion
      (that is, they have equivalent form)
```
"identical" would mean the blank nodes are the elements from the set of all blank nodes. "identical meaning" might also work.

Reply:
I agree that "identical" may sound too strong. I'm not sure about "equivalent", because this term has a precise meaning in RDF-SEMANTICS, which is not 1-1 with "isomorphic" (even though isomorphic implies equivalent).

I will go for "the same form".